coderd/debug.go

```diff
@@ -41,16 +41,22 @@ func (api *API) debugTailnet(rw http.ResponseWriter, r *http.Request) {
 // @Tags Debug
 // @Success 200 {object} healthcheck.Report
 // @Router /debug/health [get]
+// @Param force query boolean false "Force a healthcheck to run"
 func (api *API) debugDeploymentHealth(rw http.ResponseWriter, r *http.Request) {
 	apiKey := httpmw.APITokenFromRequest(r)
 	ctx, cancel := context.WithTimeout(r.Context(), api.Options.HealthcheckTimeout)
 	defer cancel()
 
-	// Get cached report if it exists.
-	if report := api.healthCheckCache.Load(); report != nil {
-		if time.Since(report.Time) < api.Options.HealthcheckRefresh {
-			formatHealthcheck(ctx, rw, r, report)
-			return
+	// Check if the forced query parameter is set.
+	forced := r.URL.Query().Get("force") == "true"
```

Comment:
I get the invalidation concept, but if it does not take too much time, maybe we can also run the health check without caching?

Reply:
> if it does not take too much time

I think this is a pretty big 'if' :-)